packages/cache/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This makes a lot of things easier. No adapter knows about a direct type (local or session storage) of cache. A facade takes care of delegating and lazily importing concrete caches.